packages/twenty-server/src/engine/twenty-orm/repository/workspace-update-query-builder.ts

```diff
@@ -250,6 +247,12 @@ export class WorkspaceUpdateQueryBuilder<
           this.authContext,
         );
 
+        this.relationNestedConfig =
```

Comment:
relationNestedConfig will be prepared for each input as they don't support batching yet